tests/Moq.Analyzers.Test/NoSealedClassMocksAnalyzerTests.cs

```diff
@@ -35,4 +35,244 @@ private void Test()
                 """,
                 referenceAssemblyGroup);
     }
+
+    public static IEnumerable<object[]> DelegateTestData()
+    {
+        return new object[][]
+        {
+            // Sealed delegates should be allowed (not trigger diagnostic)
+            ["""new Mock<SealedDelegate>();"""],
```

Comment:
SealedDelegate should not be allowed.

Reply:
Thanks for the clarification; my misunderstanding. Would you comment the code and/or tests to prevent me or another future maintainer from making the same mistake (i.e., all delegates are sealed in .NET)